### PR TITLE
GameDB: Scarface

### PR DESCRIPTION
**Pull request by pgert**
• Corrected typo errors in the description of the ScarfaceIbitHack fix.
 • Added SLES-54534, and applied the ScarfaceIbitHack fix to it since the other PAL discs have it.
 • Corrected the regioncode for SLES-54182 from PAL-F to PAL-M4.
 • Corrected the name for SLUS-21492 into its full name.

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -15644,9 +15644,18 @@ Name   = Gottlieb Pinball Classics
 Region = PAL-M5
 Compat = 5
 ---------------------------------------------
+Serial = SLUS-21111
```
Comment (FlatOutPS2) [inline bin/GameIndex.dbf]: Please keep the entries at the corect index location based on the serial.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -40995,10 +41004,6 @@ Name   = World Series of Poker - Tournament of Champions
 Region = NTSC-U
 Compat = 5
 ---------------------------------------------
-Serial = SLUS-21492
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: This was at the correct location, why did you move it ? All you had to do was rename it.

File: bin/GameIndex.dbf [hunk 3]
```diff
@@ -15659,6 +15659,16 @@ Name   = Scarface - The World is Yours
 Region = PAL-R
 ScarfaceIbitHack = 1
 ---------------------------------------------
+Serial = SLES-54271
```
Comment (FlatOutPS2) [inline bin/GameIndex.dbf]: These two (SLES-54271 and SLES-54534) are still at the wrong location.

File: bin/GameIndex.dbf [hunk 4]
```diff
@@ -16366,6 +16366,11 @@ Serial = SLES-54527
 Name   = Flushed Away
 Region = PAL-M5
 ---------------------------------------------
+Serial = SLES-54534
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: This one as well.

**Comment by FlatOutPS2** [comment]
Do the NTSC versions of Scarface not need the gamefix?

**Comment by pgert** [comment]
It's likely, but I have no way of telling.

**Comment by ssakash** [comment]
Anyone has the NTSC version to check if it's needed or not?

**Comment by FlatOutPS2** [comment]
> Anyone has the NTSC version to check if it's needed or not?

Looking at the compatibility report of the NTSC-U version it probably is necessary.

**Comment by MrCK1** [comment]
If you can't find anywhere specific that says it's needed, look for a video on YouTube somewhere.

**Comment by lightningterror** [comment]
If you can't verify that it's needed then just leave it.

**Comment by refractionpcsx2** [comment]
Is this ready to be merged now?

**Comment by FlatOutPS2** [comment]
Yeah, just merge it as it is for now. If someone confirms the other versions in the future they can be added then.

**Comment by lightningterror** [comment]
I forgot to mention this but next time please add a bit more description in the commit.
What was changed, what got fixed ...etc.

**Comment by FlatOutPS2** [comment]
I see the commits weren't squashed during the merge. :open_mouth:

**Comment by refractionpcsx2** [comment]
I didn't realise it was multiple commits, my apologies :P
